modules/log-upgrade/6x_deploying-a-clusterlogforwarder-observability-custom-resource.adoc

```diff
@@ -0,0 +1,66 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="deploying-a-clusterlogforwarder-observability-custom-resource_{context}"]
+= Deploying a ClusterLogForwarder observability custom resource
+
+Deploying a ClusterLogForwarder observability custom resource (CR) by using the {oc-first} command.
```

Comment:
This example is only if LokiStack is installed and desired to store the logs on it. If someone is not storing the logs in the Red Hat Log Store, this example is not valid. Then, it's needed to highlight this. Similar to how it's documented in the article https://access.redhat.com/articles/7089860 where it's said:

"Assuming the current stack looks like the below that represents a fully managed OpenShift Logging stack (Vector, Loki) including collection, forwarding and storage."